docs/cli.md

```diff
@@ -1082,6 +1082,65 @@ The `whoami` command returns the Viam user for an authenticated CLI session, or
 viam whoami
 ```
 
+### `auth-app`
+
+The `auth-app` command allows you to register and update your mobile or web app (created with the Viam Flutter or TypeScript [SDKs](/sdks/)) with [FusionAuth](https://fusionauth.io/) (the tool Viam uses for authentication and authorization) so that you or other users can log into your app with the same credentials they use to log into the [Viam app](https://app.viam.com).
```

Comment:
```suggestion
The `auth-app` command allows you to register and update your web or mobile application (created with the Viam Flutter or TypeScript [SDKs](/sdks/)) with [FusionAuth](https://fusionauth.io/) (the tool Viam uses for authentication and authorization) so that you or other users can log into your app with the same credentials they use to log into the [Viam app](https://app.viam.com).
```
This would probably stop linter from complaining

Reply:
Ooh clever! :D